starter-kits/nextjs-starter-kit/primitives/EmojiPicker/EmojiPicker.module.css

```
@@ -0,0 +1,87 @@
+.root[frimousse-root] {
+  display: flex;
+  flex-direction: column;
+  width: fit-content;
+  height: 352px;
+  background: light-dark(#fff, #171717);
+  isolation: isolate;
+}
+
+.root [frimousse-search] {
+  position: relative;
+  z-index: 10;
+  appearance: none;
+  padding: 8px 10px;
+  background: light-dark(#f5f5f5, #262626);
+  border-radius: 6px;
+  font-size: 14px;
+  height: var(--space-13);
+  max-width: 175px;
+}
+
+.root [frimousse-search]:focus {
+  outline-color: var(--color-accent);
+}
+
+.root [frimousse-viewport] {
+  position: relative;
+  flex: 1;
+  outline: none;
+}
+
+.root [frimousse-loading] {
```

The empty state is currently unstyled

```suggestion
.root [frimousse-loading],
.root [frimousse-empty] {
```

---

Thanks! You could add this to [the CSS snippet here](https://frimousse.liveblocks.io/#usage) too.